web/crux-ui/e2e/with-login/resource-copy.spec.ts

```diff
@@ -0,0 +1,344 @@
+    const projectName = 'delete-default-check-instances'
+    const prefix = projectName
+
+    const projectId = await createProject(page, projectName, 'versioned')
+    const defaultVersionName = 'default-version'
+    const defaultVersionId = await createVersion(page, projectId, defaultVersionName, 'Rolling')
+    await addImageToVersion(page, projectId, defaultVersionId, NGINX_TEST_IMAGE_WITH_TAG)
+    await addDeploymentToVersion(page, projectId, defaultVersionId, DAGENT_NODE, prefix)
+
+    const newVersionId = await createVersion(page, projectId, 'new-version', 'Rolling')
+
+    await page.goto(versionUrl(projectId, defaultVersionId))
+
+    await deleteVersion(page, projectId, defaultVersionId)
+
+    await page.goto(projectUrl(projectId))
+
+    expect(page.locator(`:has-text("${defaultVersionName}")`)).toHaveCount(0)
+
+    await page.goto(versionUrl(projectId, newVersionId))
+
+    await page.locator('button:has-text("Deployments")').click()
+
+    const deploymentsTabelBody = await page.locator('.table-row-group')
+    const deploymentRows = await deploymentsTabelBody.locator('.table-row')
+
+    await expect(deploymentRows).toHaveCount(1)
+
+    const viewDeploymentButton = await page.waitForSelector(`[src="/eye.svg"]`)
+    await viewDeploymentButton.click()
+
+    const instancesTabelBody = await page.locator('.table-row-group')
+    const instanceRows = await instancesTabelBody.locator('.table-row')
+    await expect(instanceRows).toHaveCount(1)
+  })
+
+  test('should not affect the instance config of the deployment of a new version', async ({ page }) => {
+    const projectName = 'delete-default-check-instance-config'
+    const prefix = projectName
+
+    const projectId = await createProject(page, projectName, 'versioned')
+    const defaultVersionName = 'default-version'
+    const defaultVersionId = await createVersion(page, projectId, defaultVersionName, 'Rolling')
+    await addImageToVersion(page, projectId, defaultVersionId, NGINX_TEST_IMAGE_WITH_TAG)
+    const { id: defaultDeploymentId } = await addDeploymentToVersion(
+      page,
+      projectId,
+      defaultVersionId,
+      DAGENT_NODE,
+      prefix,
+    )
+
+    await page.goto(deploymentUrl(defaultDeploymentId))
+
+    const instancesTableBody = await page.locator('.table-row-group')
+    const instancesRows = await instancesTableBody.locator('.table-row')
+
+    await expect(instancesRows).toHaveCount(1)
+    await expect(page.locator('div.table-cell:has-text("nginx")').first()).toBeVisible()
+
+    const settingsButton = await page.waitForSelector(`[src="/settings.svg"]:right-of(:text("nginx"))`)
+    await settingsButton.click()
+
+    const sock = waitSocket(page)
+    await page.waitForSelector(`h2:has-text("Container")`)
+    const ws = await sock
+    const wsRoute = deploymentWsUrl(defaultDeploymentId)
+
+    const internal = '1000'
+    const external = '2000'
+    await addPortsToContainerConfig(page, ws, wsRoute, internal, external)
```

Comment:
This fails because `addPortsToContainerConfig` uses `wsPatchSent` which waits for `WS_TYPE_PATCH_IMAGE` but in this case it's an instance patch so it never receives the expected frame :/